packages/app/src/components/SourceForm.tsx

```diff
@@ -489,7 +495,18 @@ export function LogTableModelForm(props: TableModelProps) {
           />
         </FormRow>
         <Divider />
-        <HighlightedAttributeExpressionsFormRow {...props} />
+        <HighlightedAttributeExpressionsFormRow
```

Comment:
Are these supposed to show up on Logs and Metrics? I could only get them to work under Traces.